ArduCopter/ArduCopter.cpp

```diff
@@ -152,7 +152,7 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
     SCHED_TASK_CLASS(AP_InertialSensor,    &copter.ins,                 periodic,       400,  50),
     SCHED_TASK_CLASS(AP_Scheduler,         &copter.scheduler,           update_logging, 0.1,  75),
 #if RPM_ENABLED == ENABLED
-    SCHED_TASK(rpm_update,            10,    200),
+    SCHED_TASK(rpm_update,            40,    200),
```

Comment:
This will lead to the RPM data being logged at 40hz as well.  Perhaps that's OK but I just want to be sure you're aware.

Reply:
Yes, we are aware of this. Discussed it with tridge in the early stages of governor development. He said it is a very lost cost change from a resources standpoint. The five-element mode filter in the rpm library sort of slows things down a bit and we're reducing the latency as much as possible to get real-time rpm reading here.